src/types.ts

```diff
@@ -1,11 +1,37 @@
-export { Request as ExpressRequest } from 'express';
+type Timestamp = number;
 
-export enum Products {
-  jira = 'jira',
-  confluence = 'confluence',
-  bitbucket = 'bitbucket',
+export const ContextQsh = 'context-qsh';
+
+/**
+ * Represents an extensible payload of a Connect JWT.
+ */
+export interface ConnectJwt {
+  sub: string;
+  qsh?: string | typeof ContextQsh;
```

Comment:
Does this type should be read as "arbitrary sting or 'context-qsh' value"? My Typescript knowledge limps here a bit.

Reply:
You are correct. This does not help much the compiler but gives a hint to the developer of a possible content.